drivers/unix/ip_unix.cpp

```diff
@@ -129,7 +129,7 @@ void IP_Unix::_resolve_hostname(List<IP_Address> &r_addresses, const String &p_h
 		IP_Address ip = _sockaddr2ip(next->ai_addr);
 		if (!r_addresses.find(ip)) {
 			r_addresses.push_back(ip);
-        }
+		}
```

Comment:
It would be better if this fix was part of the first commit, so we don't go back and forth and save two needless diffs in the history. Or both commits could be squashed into one, as you prefer.

Reply:
I've squashed them into a single commit.